graphics/macgui/macfontmanager.cpp

```diff
@@ -640,6 +648,41 @@ int MacFontManager::registerFontName(Common::String name, int preferredId) {
 	return id;
 }
 
+int MacFontManager::registerTTFFont(Graphics::TTFMap *ttfList) {
+	int ret = 1;
+
+	for (int i = 0; i < 4; i++) {
+		int id = 1000;
```

Comment:
Why is it hardcoded? Does it mean that I cannot register two fontmaps?

Reply:
fixed